core/domain/html_cleaner_test.py

```diff
@@ -27,6 +27,17 @@ def setUp(self):
         super(HtmlCleanerUnitTests, self).setUp()
         self.longMessage = True
 
+    def test_whitelisted_a_tags(self):
```

Comment:
Add tests for non-a tags, now that the definition of your function is broader.

Reply:
I've added a few more tests and I believe all the needed tests are now covered Thanks!

Reply:
Hi @seanlip & @apb7,
I've updated it learning from the examples in the doc you mentioned.
I've renamed it to filter_tag and it made available/callable for all the tags.
PTAL Thanks!

Reply:
Hi @NishealJ, I think you wrote this in the wrong comment thread, I'm copying it above.

Reply:
Thanks